tools/build-a-node

```diff
@@ -91,13 +91,51 @@ if [[ -z "$version" ]]; then
     echo "build-a-node: fatal error: no 'version' given" >&2
     exit 1
 fi
+
+# Before we checkout the branch, copy off the most recent CA header file.
+tfile=$(mktemp)
+cp src/node_root_certs.h $tfile
```

Comment:
Should we explicitly check out master here just to be sure we're on the right branch for grabbing the header?

Reply:
build-a-node (builds one version) does:
- `cp -PR $TOP/build/src $buildDir/src`  where $TOP/build/src is master (see the nodesrc: target in Makefile)
- `cd $buildDir/src` (i.e. PWD == copy of `master`)
- (NEW) copies node_root_certs.h to /tmp
- checks out appropriate version
- (NEW) replaces node_root_certs.h from one in /tmp with appropriate build-specific modifications
- builds